src/panels/config/tags/ha-config-tags.ts

```diff
@@ -193,6 +195,9 @@ export class HaConfigTags extends SubscribeMixin(LitElement) {
         .noDataText=${this.hass.localize("ui.panel.config.tags.no_tags")}
         hasFab
       >
+        <mwc-icon-button slot="toolbar-icon" @click=${this._showHelp}>
+          <ha-svg-icon .path=${mdiHelpCircle}></ha-svg-icon>
```

Comment:
is this imported?

Reply:
As far as I can tell all the necessary imports are there. I added mwc-icon-button, but it doesn't look like ha-svg-icon import is necessary for some reason. I'm not super familiar with this system yet though

Reply:
it is needed, but probably already is imported by a parent. You should add it though if the order ever changes it breaks.